packages/components/src/internal/components/permissions/EffectiveRolesList.tsx

```diff
@@ -6,17 +6,22 @@ import React from 'react';
 import { Map, List } from 'immutable';
 
 import { SecurityAssignment, SecurityPolicy, SecurityRole } from './models';
+import {Col, Row} from "react-bootstrap";
+import {User} from "../base/models/User";
+import {AppURL} from "../../url/AppURL";
 
 interface Props {
+    currentUser: User;
     policy?: SecurityPolicy;
     rolesByUniqueName?: Map<string, SecurityRole>;
     rootPolicy?: SecurityPolicy;
+    showLinks?: boolean;
     userId: number;
 }
 
-export class EffectiveRolesList extends React.PureComponent<Props, any> {
+export class EffectiveRolesList extends React.PureComponent<Props> {
     render() {
-        const { userId, policy, rootPolicy, rolesByUniqueName } = this.props;
+        const { userId, policy, rootPolicy, rolesByUniqueName, currentUser, showLinks = true } = this.props;
```

Comment:
We aren't showing the Effective Roles on the modal.  Is that intentional?

Reply:
It was. The UserDetailsPanel didn't always have the security policy in all of the contexts. But I just fixed that so that we now do show the Effective Roles in the modal.